lib/game_of_life_web/live/universe_live.ex

```diff
@@ -1,7 +1,9 @@
 defmodule GameOfLifeWeb.UniverseLive do
   use Phoenix.LiveView
+
```

Comment:
no problem with adding this in but are there usually blank lines between `use` and `alias`es?

Reply:
there isn't a standard for this.  I prefer to separate `import` `use` and `alias` for readability

Reply:
I like that

Reply:
sounds good to me! I don't have a preference either way, just wanted to make sure it wasn't an oopsy-add